python/mxnet/foo/nn/conv_layers.py

```diff
@@ -158,6 +158,22 @@ class Conv1D(_Conv):
         see Initializer.
     bias_initializer: Initializer for the bias vector
         see Initializer.
+
+
+    Input Shape:
+        This depends on the `layout` parameter. Input is 3D array of shape
+        (batch_size, channel, width) if `layout` is `NCW`.
```

Comment:
channel is not defined. In the parameters section these are in_filters and filters. we should mention filters/in_filters is the number of output/input channels somewhere